roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -37,3 +37,43 @@
 - set_fact:
     docker_options: "{{ openshift.docker.options | default(omit) }}"
   when: not openshift.docker.hosted_registry_insecure | default(False) | bool
+
+# Avoid docker 1.9 when installing origin < 1.2 or OSE < 3.2 on RHEL/Centos and
+# See: https://bugzilla.redhat.com/show_bug.cgi?id=1304038
+- name: Set docker defaults
+  set_fact:
+    docker_version: ''
+    docker_downgrade: False
```

Comment:
Instead of setting docker_downgrade to False by default, you can no set it and then test later on:
```
docker_downgrade is defined
```